libs/base/src/lib/context/AccountProvider.tsx

```diff
@@ -126,6 +134,11 @@ const OnboardProvider: FC<{
               setAddress(account.toLowerCase())
             }
           },
+          ens: ens => {
+            if (ens) {
```

Comment:
```suggestion
            if (!!ens.name) {
```

Reply:
Decided to go another way in the end.

`ens` can be undefined so it's not a good idea to check a property of `ens` without knowing whether it's undefined or not.

Reply:
Perfect, looks good – appreciate the PR!